code/datums/components/twohanded.dm

```diff
@@ -140,12 +140,8 @@
 
 	if(auto_wield)
 		UnregisterSignal(user, COMSIG_MOB_SWAP_HANDS)
-	if(require_twohands)
-		unwield(TRUE)
-	if(wielded)
+	if(require_twohands || wielded)
 		unwield()
```

Comment:
What did the TRUE do and was it important?

Reply:
This changes the behaviour of unwield by making it not display a message when you unwield a twohanded item.

Reply:
it's true by default